modules/sbo-connecting-spring-petclinic-sample-app-to-postgresql-database-service-power-z.adoc

```diff
@@ -11,25 +12,24 @@ To connect the sample application to the database service, you must create a `Se
 +
 [source,terminal]
 ----
-$ oc apply -f - << EOD
+$ oc apply -n my-petclinic -f - << EOD
 ---
 apiVersion: binding.operators.coreos.com/v1alpha1
 kind: ServiceBinding
 metadata:
-    name: spring-petclinic-rest
-    namespace: my-postgresql
+    name: spring-petclinic-pgcluster
 spec:
-    services: <1>
+  services: <1>
     - group: postgresql.dev4devs.com
       version: v1beta1
       kind: Database <2>
       name: sampledatabase
       version: v1alpha1
```

Comment:
the correct version `v1alpha1` is already specified so `version: v1beta1` should be removed. @johusta  could you confirm that the version is `version: v1alpha1` on Z as well. Thanks.
```suggestion
    - group: postgresql.dev4devs.com
      kind: Database <2>
      name: sampledatabase
      version: v1alpha1
```

Reply:
Good catch Pravin I added comment to the google doc